src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -2180,6 +2180,11 @@ void MVN::MVNJitExecutor::mvn_pln(const uint8_t* src_data, uint8_t* dst_data, co
     size_t C2 = C1 * D;
     size_t C3 = C2 * C;
 
+#if (OV_THREAD == OV_THREAD_OMP)
+    const auto origin_nested_levels = get_max_nested_levels();
+    set_max_nested_levels(origin_nested_levels + 1);
```

Comment:
Agreed offline that nesting settings shoudn't be changed as long as it could ruin the perf.

Reply:
Removed